src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -910,21 +906,6 @@ private static ControlledResourceCommonFields createCommonFields(
             .accessScope(createParams.accessScope)
             .managedBy(ManagedBy.USER);
 
```

Comment:
Looks like the following aren't used any more, can you delete them?

- CreateResourceParams.accessScope
- ControlledResourceCommonFields.accessScope

Reply:
They are used in a couple of spots. Might need a separate work item to clean those up.
The ControlledResourceCommonFields comes from a different library.

Reply:
Thanks. Sorry new to this code, CreateResourceParams.accessScope is still used.